### PR TITLE
DSD-1693: responsive size bug for heading5

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1693](https://jira.nypl.org/browse/DSD-1693)

## This PR does the following:

- Fixes the mobile font size for the `"heading5"` variant in the `Heading` component.
- NOTE: This bug is only present in the React 17 version of DS..

## How has this been tested?

- local Storybook

## Accessibility concerns or updates

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Heading/headingChangelogData.ts [hunk 1]
```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Bug Fix",
+    affects: ["Styles"],
+    notes: ['Fixed the mobile font size for the "heading5" variant.'],
```
Comment (isastettler) [inline src/components/Heading/headingChangelogData.ts]: Nit: 'Fixes the desktop font size...'

**Review by isastettler — COMMENTED** [review]
Looks good. Thanks for adding this so quickly.

File: src/components/Heading/headingChangelogData.ts (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Heading/headingChangelogData.ts]: No, that's not a `nit`.

**Review by isastettler — APPROVED** [review]
Looks good to me.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**